packages/medusa/src/utils/authenticate-middleware.ts

```diff
@@ -67,12 +67,27 @@ export const authenticate = (
       }
     }
 
-    if (authUser) {
+    const isMedusaScope =
```

Comment:
I think after you rebase you'll see that this code was added already by Philip 😄